drop-cap/library/src/main/java/com/novoda/dropcap/DropCapView.java

```diff
@@ -91,7 +98,30 @@ private void applyCustomAttributes(Context context, AttributeSet attrs) {
         }
     }
 
-    public void setDropCapFontType(String fontPath) {
+    public void setNumberOfDropCaps(int numberOfDropCaps) {
+        this.numberOfDropCaps = numberOfDropCaps;
+
+        String text = dropCapText + copyText;
+        setText(text);
+    }
+
+    public void setText(@Nullable String text) {
+        if (enoughTextForDropCap(text)) {
+            dropCapText = String.valueOf(text.substring(0, numberOfDropCaps));
+            copyText = String.valueOf(text.subSequence(dropCapText.length(), text.length()));
```

Comment:
I did this more to avoid the pain of experienced when putting the `numberOfDropCaps` in again in the future 😆  when I was adding `numberOfDropCaps` I needed to go through and take out `0s` and `1s` I wanted to avoid that by using the actual text as it is something that will always be present.